R/cchart.R

```diff
@@ -299,6 +302,34 @@ CChart <- function(chart.type, x, small.multiples = FALSE,
     result
 }
 
+addWarning <- function(x, chart.type, small.multiples, has.annotations)
```

Comment:
In this function there is inconsistent use of Microsoft and Powerpoint to refer to the same thing, Microsoft Powerpoint. (e.g. Powerpoint is used in the last else if block but Microsoft is used everywhere else. Looking at the Displayr interface, it would appear Powerpoint would be the better choice to use throughout?